components/prism-llvm.js

```diff
@@ -0,0 +1,12 @@
+(function(Prism) {
+	Prism.languages.llvm = {
+		'comment': /;.*/,
+		'string': /"[^"]*"/,
+		'variable': /[%@!#](?:(?:[-a-zA-Z$._]|\\[a-fA-F0-9]{2})(?:[-a-zA-Z$._0-9]|\\[a-fA-F0-9]{2})*|[0-9]+)/,
+		'label': /(?:[-a-zA-Z$._]|\\[a-fA-F0-9]{2})(?:[-a-zA-Z$._0-9]|\\[a-fA-F0-9]{2})*:/,
+		'type': /\b(?:void|label|token|metadata|float|double|half)\b/,
```

Comment:
1. Please sort this list to improve the readability.
2. Aren't there many types missing? I mean all of the [integer types](https://llvm.org/docs/LangRef.html#integer-type) are highlighted as `keyword`, a few [floating-point types](https://llvm.org/docs/LangRef.html#floating-point-types) are missing, and [this thing](https://llvm.org/docs/LangRef.html#x86-mmx-type) is also missing.
3. `type` isn't a colored by most themes, so please give it an alias `class-name`.